v2/azure-arm.yaml

```diff
@@ -2379,7 +2379,7 @@ objectModelConfiguration:
         $nameInNextVersion: MySQLServerSku_STATUS
       ServerProperties:
         SourceServerResourceId:
-          $referenceType: simple # Actually, this *IS* a resource id, but we want to avoid breaking changes so we're fibbing here
+          $referenceType: arm+compat
```

Comment:
Do we actually want to change this?

This is an old version (2021-05-01). It was already supported in the newer versions. I guess this is safe to do, but I thought that the objective for arm+compat was especially targeted at the versions we couldn't fix (because there was no new version to fix it in), which is not the case here.

Reply:
Agreed. Removed.